src/Exchange_Rates.json

```diff
@@ -0,0 +1,23 @@
+{
+  "success": true,

```

Comment:
Why do we need this?

Reply:
We don't need it and it is not really used anywhere as well. It is just another form of 200 OK status returned by the API as JSON, **I can remove this**. Thanks.

Reply:
Yes, let's remove anything unnecessary.